.github/workflows/release_mac.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
       with:
         python-version: ${{ matrix.python-version }}
+        freethreaded: ${{ endsWith(${{ matrix.python-version }}, 't') }}
```

Comment:
why does that have to be added? I don't see it at https://github.com/actions/setup-python

Reply:
See the document `https://github.com/actions/setup-python/blob/5db1cf9a59fb97c40a68accab29236f0da7e94db/docs/advanced-usage.md?plain=1#L80`

Reply:
but we have in every case a specific version?

```
Free threaded Python is only available starting with the 3.13 release.

```yaml
steps:
- uses: actions/checkout@v4
- uses: actions/setup-python@v5
  with:
    python-version: '3.13t'
- run: python my_script.py
```

Note that the **t** suffix is not `semver` syntax. If you wish to specify a range, you must use the `freethreaded` input instead of the `t` suffix.

```yaml
steps:
- uses: actions/checkout@v4
- uses: actions/setup-python@v5
  with:
    python-version: '>=3.13'
    freethreaded: true
- run: python my_script.py
```